addOns/exim/src/main/javahelp/help/contents/exim.html

```diff
@@ -7,11 +7,15 @@
 </TITLE>
 </HEAD>
 <BODY>
+
+<H1>>Copy URLs to Clipboard</H1>
+A context menu item to Copy URLs to the system clipboard.
+
 <H1>Save Selected Entries as HAR (HTTP Archive File)</H1>
 A context menu item to save the selected HTTP messages in HAR format.
 
 <H1>Save Raw Message</H1>
-Provides a context menu to save content of HTTP messages as binary.
+Provides a context menu to save content of HTTP messages as text.
```

Comment:
It would be better to keep as it was, the content is not "plain text" (e.g. can have null characters; malformed bytes…).

Reply:
I've reverted this and added a sentence clarifying.